ruby/rspec/tutorial.md

```diff
@@ -0,0 +1,155 @@
+# RSpec Tutorial/Guide
+
+RSpec is a testing tool for Ruby, it is the most frequently used testing library in Ruby applications, where other tools in other languages were inspired by like jasmine and mocha.  This tutorial will aim to steer you into the path of testing and show you the different ways of testing.
```

Comment:
`where other tools in other languages were inspired by like jasmine and mocha`

This sentence sounded a bit confusing to me. Maybe something like `and it was the source of inspiration for testing tools in other languages like jasmine and mocha`.